install/install_1-1_postgresql.php

```diff
@@ -36,41 +36,6 @@ public function __construct($db, $table)
 	public function create_functions()
 	{
 		$this->db->query('', '
-		CREATE OR REPLACE FUNCTION FROM_UNIXTIME(integer) RETURNS timestamp AS
-			\'SELECT timestamp \'\'epoch\'\' + $1 * interval \'\'1 second\'\' AS result\'
-		LANGUAGE \'sql\';');
-
-		$this->db->query('', '
-		CREATE OR REPLACE FUNCTION IFNULL (text, text) RETURNS text AS
-			\'SELECT COALESCE($1, $2) AS result\'
-		LANGUAGE \'sql\';');
-
-		$this->db->query('', '
-		CREATE OR REPLACE FUNCTION IFNULL (int4, int4) RETURNS int4 AS
-			\'SELECT COALESCE($1, $2) AS result\'
-		LANGUAGE \'sql\';');
-
-		$this->db->query('', '
-		CREATE OR REPLACE FUNCTION IFNULL (int8, int8) RETURNS int8 AS
-			\'SELECT COALESCE($1, $2) AS result\'
-		LANGUAGE \'sql\';');
-
-		$this->db->query('', '
-		CREATE OR REPLACE FUNCTION IFNULL (character varying, character varying) RETURNS character varying AS
-			\'SELECT COALESCE($1, $2) AS result\'
-		LANGUAGE \'sql\';');
-
-		$this->db->query('', '
-		CREATE OR REPLACE FUNCTION IFNULL (character varying, boolean) RETURNS character varying AS
-			\'SELECT COALESCE($1, CAST(CAST($2 AS int) AS varchar)) AS result\'
-		LANGUAGE \'sql\';');
-
-		$this->db->query('', '
-		CREATE OR REPLACE FUNCTION IFNULL (int, boolean) RETURNS int AS
-			\'SELECT COALESCE($1, CAST($2 AS int)) AS result\'
-		LANGUAGE \'sql\';');
-
-		$this->db->query('', '
```

Comment:
Would you mind leaving them for 1.1?
I'm a bit too lazy to fix my addons if they work. :stuck_out_tongue: :wink: